psi-ms.obo

```diff
@@ -23617,6 +23621,318 @@ relationship: has_metric_category MS:4000008 ! ID based metric
 relationship: has_metric_category MS:4000016 ! retention time metric
 relationship: has_value_type xsd:float ! The allowed value-type for this CV term
 
+[Term]
+id: MS:4000151
+name: MsQuality
+def: "MsQuality – an interoperable open-source package for the calculation of standardized quality metrics of mass spectrometry data." [DOI:10.1101/2023.05.12.540477, https://github.com/tnaake/MsQuality/]
+is_a: MS:1001456 ! analysis software
+
+[Term]
+id: MS:4000152
+name: MS2 precursor median m/z of identified quantification data points
+def: "Median m/z value for MS2 precursors of all quantification data points after user-defined acceptance criteria are applied. These data points may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the metadata or analysis methods section of the recording file for the respective run. In case of multiple acceptance criteria (FDR) available in proteomics, PSM-level FDR should be used for better comparability." [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+relationship: has_metric_category MS:4000022 ! MS2 metric
+is_a: MS:4000020 ! ion source metric
+
+[Term]
+id: MS:4000153
+name: interquartile RT period for identified quantification data points
+def: "The interquartile retention time period, in seconds, for all quantification data points after user-defined acceptance criteria are applied over the complete run. Longer times indicate better chromatographic separation. These data points may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the metadata or analysis methods section of the recording file for the respective run. In case of multiple acceptance criteria (FDR) available in proteomics, PSM-level FDR should be used for better comparability." [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000017 ! chromatogram metric
+synonym: "C-2A"  RELATED [PMID:19837981]
+
+[Term]
+id: MS:4000154
+name: rate of the interquartile RT period for identified quantification data points
+def: "The rate of identified quantification data points for the interquartile retention time period, in identified quantification data points per second. Higher rates indicate efficient sampling and identification. These data points may be for example XIC profiles, isotopic pattern areas, or reporter ions (see MS:1001805). The used type should be noted in the metadata or analysis methods section of the recording file for the respective run. In case of multiple acceptance criteria (FDR) available in proteomics, PSM-level FDR should be used for better comparability." [PSI:MS]
```

Comment:
**suggestion:** The unit could also be explicitly encoded using `relationship: has_units UO:0000106 ! hertz`.